authors/apps/authentication/models.py

```diff
@@ -8,14 +8,13 @@
 
 
 class UserManager(BaseUserManager):
-    """
-    Django requires that custom users define their own Manager class. By
-    inheriting from `BaseUserManager`, we get a lot of the same code used by
-    Django to create a `User` for free. 
 
-    All we have to do is override the `create_user` function which we will use
-    to create `User` objects.
-    """
+    # Django requires that custom users define their own Manager class. By
```

Comment:
Use doc string for this

Reply:
The hash comments were already there in the codebase. Actually, I tried to change them to doc strings and got merge conflicts after Kasule's PR was merged
So I reverted them, that's why they are in red

Reply:
You could just rebase against `develop` and resolve the conflicts before pushing.